engine/pkg/meta/model/config.go

```diff
@@ -31,6 +31,9 @@ const (
 	StoreTypeEtcd = "etcd"
 	// StoreTypeSQL is the store type string for SQL
 	StoreTypeSQL = "sql"
+	// StoreTypeSQLite is the store type string for SQLite
+	// NOTE: ONLY for test
+	StoreTypeSQLite = "sqlite"
```

Comment:
Do we use any feature that is incompatible with MySQL?

Reply:
I'm not quite sure.
`sqlite` has some tests in  https://github.com/pingcap/tiflow/blob/master/engine/pkg/orm/mock_test.go. It has cover most of our use cases.

Reply:
But the `sqlite` backend uses `sqlite driver` actually, so It can't verify the `compatibility with MySQL` actually.
I don't have better scheme to use an embeded mysql-compatible db for flexible unit test.

Do you have any suggestion?

Reply:
MySQL and SQLite both provide SQL interfaces. So I guess the `sql` type can contain the `sqlite` type. The only difference is the sql driver we are using. I see a new `sqlite` type added here, so I'm curious if we are using some features that are supported by SQLite but not by MySQL.

Reply:
Only for test. 
`sql.DB` is not compatible with `sqlite`, so not all the abstract can be used for `sqlite` and `mysql`.
So I make a new conn type for mock test.

Reply:
You are right.
I will move the `sqlite` connection intialization to the `ClientConn` layer and reuse the `sql.DB`.
But we still need `sqlite` type to indicate different driver.

Reply:
If we indeed need `sqlite` type, I suggest renaming `StoreTypeSQL` to `StoreTypeMySQL`, because the semantics of `sql` contains `MySQL` and `SQLite`.

Reply:
`MySQL` is a too specific DB name, and I think it may confuses user that only `mysql` can be the metastore.

Reply:
We might support different databases, like `MySQL`, `PostgreSQL` or `SQLite`. Although they have a similar  `SQL` interface, we still need to know which `SQL` driver we can use. Usually, [`DSN`](https://en.wikipedia.org/wiki/Data_source_name) is used to represent a data source, the driver is placed at the beginning as a scheme.

Reply:
Done. Has changed `StoreTypeSQL` to `StoreTypeMySQL`.